brownfield/firewall.go

```diff
@@ -42,7 +42,7 @@ func startFirewallMapping(cs *utils.HTTPConcertoservice, rules []*discovery.Fire
 		err = fmt.Errorf("server responded with %d code: %s", status, string(body))
 		return
 	}
-	responseData := &fw.FirewallProfile{}
+	responseData := &types.Firewall{}
```

Comment:
I'd rather keep calling the models FirewallProfile and Rules (or maybe FirewallRules). I think the Policy terminology was introduced in the context of the application of firewall on Unix systems

Reply:
Well, there is a conflict with naming. Policy, in some way has been used in order to solve the conflict with another FirewallProfile/Rule context defined for user mode. 

FirewallProfile is currently used. FirewallRule could be a good one.

I'm going to re-evaluate how to clarify, and separate this adequately keeping in mind the "Firewall" User and Server contexts